src/js/components/awardv2/idv/activity/chart/ActivityChartBar.jsx

```diff
@@ -5,13 +5,14 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import cx from 'classnames';
```

Comment:
Introducing a new dependency just to handle the class names seems unnecessary - we could either pass a boolean prop indicating whether this is an obligated bar, or the class name itself as a string.

Reply:
Thank you @ebdabbs. Code for all comments have been updated.